src/shell/shell_cmds.cpp

```diff
@@ -740,6 +741,74 @@ void DOS_Shell::CMD_DIR(char * args) {
 	dos.dta(save_dta);
 }
 
+void DOS_Shell::CMD_LS(char *args)
+{
+	HELP("LS");
+
+	RealPt save_dta = dos.dta();
```

Comment:
Minor: `save_dta` sounds more like an action.  Suggest: `saved_dta`, `original_dta`, `prior_dta`.

Reply:
Good point. I copied this line from DIR, but could've improved on naming. DTA means [Disk Transfer Area](https://stanislavs.org/helppc/dta.html) btw.